troposphere/asg-elb-dns.py

```diff
@@ -1,14 +1,26 @@
 from troposphere import Join, Output, GetAtt
 from troposphere import Parameter, Ref, Template
-from troposphere.autoscaling import AutoScalingGroup, Tag
-from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, AutoScalingCreationPolicy, ResourceSignal
+from troposphere.autoscaling import AutoScalingGroup, Tag, ScalingPolicy
+from troposphere.cloudwatch import Alarm, MetricDimension
+from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, \
+    AutoScalingCreationPolicy, ResourceSignal
 import troposphere.elasticloadbalancing as elb
 from troposphere.route53 import RecordSetType
+import sys
 
 __author__ = 'Jose Armesto'
 
 
 def generate_cloudformation_template():
+    enable_elb = sys.argv[1]
+    input_scaling_policies = eval(sys.argv[2])
+    input_alarms = eval(sys.argv[3])
+
+    if enable_elb == 'True':
```

Comment:
Would I get any benefit from using Click here? I'm not sure this is the place for Click.

I think the first suggestion wouldn't work when enable_elb is "False"
```
$ python
Python 2.7.13 (default, Feb  7 2017, 22:08:53)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> enable_elb = 'False'
>>> print(bool(enable_elb))
True
```

But this second suggestion is better that the solution I have
```
enable_elb = enable_elb == 'True'
```

So I will use the latter!